src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -136,6 +113,28 @@ public void Start()
             this.State = CoreNodeState.Running;
         }
 
+        private void CreateConfigFile(NodeConfigParameters configParameters = null)
+        {
+            Directory.CreateDirectory(this.runner.DataFolder);
+
+            configParameters = configParameters ?? new NodeConfigParameters();
+            configParameters.SetDefaultValueIfUndefined("regtest", "1");
+            configParameters.SetDefaultValueIfUndefined("rest", "1");
+            configParameters.SetDefaultValueIfUndefined("server", "1");
+            configParameters.SetDefaultValueIfUndefined("txindex", "1");
+            if (!this.CookieAuth)
+            {
+                configParameters.SetDefaultValueIfUndefined("rpcuser", this.creds.UserName);
+                configParameters.SetDefaultValueIfUndefined("rpcpassword", this.creds.Password);
+            }
+
+            configParameters.SetDefaultValueIfUndefined("printtoconsole", "1");
```

Comment:
I don't see the ports here. Is it intentional?

Reply:
yes, it is now happening in the constructor line 60
you can try to see what happens by running the tests in CustomNodeBuilderTests.cs
![image](https://user-images.githubusercontent.com/4638821/42493187-d3155a36-8413-11e8-99b0-f21f5270731b.png)